python/orca/test/bigdl/orca/learn/ray/pytorch/test_estimator_pyspark_backend.py

```diff
@@ -661,19 +661,28 @@ def test_optional_optimizer(self):
         shards = rdd.mapPartitions(lambda iter: chunks(iter, 5)).map(lambda x: {"x": np.stack(x)})
         shards = SparkXShards(shards)
 
-        estimator = Estimator.from_torch(model=lambda config: IdentityNet(),
-                                         loss=nn.BCELoss(),
-                                         metrics=Accuracy(),
-                                         config={"lr": 1e-2},
-                                         workers_per_node=2,
-                                         backend="spark",
-                                         sync_stats=False,
-                                         log_level=logging.DEBUG)
+        try:
+            trainer = get_estimator(model_fn=lambda config: IdentityNet())
+            path = "/tmp/optimizer_model.pth"
+            trainer.save(path)
+            trainer.shutdown()
+
+            estimator = Estimator.from_torch(model=lambda config: IdentityNet(),
+                                             loss=nn.BCELoss(),
+                                             metrics=Accuracy(),
+                                             config={"lr": 1e-2},
+                                             workers_per_node=2,
+                                             backend="spark",
+                                             sync_stats=False,
```

Comment:
remove these after https://github.com/intel-analytics/BigDL/issues/7161#issuecomment-1370599972 is fixed?